definitions/ext-rf_scanner/summary_metrics.yml

```diff
@@ -1,14 +1,15 @@
-
 latency:
+  goldenMetric: latency
   title: Latency (ms)
```

Comment:
Milliseconds!

Reply:
This can stay. why?
the golden metric definition is exactly the same as the summary metric. So This is not changing any behaviour in the UI. hence, we can fix it later.